docs/sources/flow/reference/components/otelcol.exporter.loadbalancing.md

```diff
@@ -301,3 +395,565 @@ otelcol.exporter.loadbalancing "default" {
     }
 }
 ```
+
+### DNS resolver
+
+When configured with a `dns` resolver, `otelcol.exporter.loadbalancing` will do a DNS lookup 
+on regular intervals. Spans will then be exported to the addresses which the DNS lookup had returned.
+
+```river
+otelcol.exporter.loadbalancing "default" {
+    resolver {
+        dns {
+            hostname = "grafana-agent-traces-sampling.grafana-cloud-monitoring.svc.cluster.local"
+            port     = "34621"
+            interval = "5s"
+            timeout  = "1s"
+        }
+    }
+    protocol {
+        otlp {
+            client {}
+        }
+    }
+}
+```
+
+Below is an example Kubernetes configuration which sets up two sets of Agents:
+* A pool of "load balancer" Agents:
```

Comment:
Nit: I would consider calling them `load-balancing Agents` and `backing Agents` to avoid the quotation marks... gives the impression it's a temporary name, but in fact it seems like a perfectly good name.

Reply:
TBH I don't like word "backing", but I use it here because I have heard it mentioned in OTel discussions. I think "sampling" makes it more clear what the Agents are doing. WDYT? I'll also ask the OTel experts if there is a convention we must follow.